### PR TITLE
Update seemingly incorrect function names from user to order; Update caniuse-lite

### PR DESCRIPTION
**Pull request by rubengmurray**
- It looks like these exported function names are incorrect, the paths all logically point to interacting with orders by their unique id
- Update caniuse-lite after update warning running tests locally

### CODE REVIEW

File: example-application/business-logic/order-service.js [hunk 1]
```diff
@@ -47,12 +47,12 @@ module.exports.addOrder = async function (newOrder) {
...
 
-module.exports.deleteUser = async function (userId) {
-  return await new OrderRepository().deleteOrder(userId);
+module.exports.deleteOrder = async function (id) {
+  return await new OrderRepository().deleteOrder(id);
 };
 
-module.exports.getUser = async function (userId) {
-  return await new OrderRepository().getOrderById(userId);
+module.exports.getOrderById = async function (id) {
```
Comment (rluvaton) [inline example-application/business-logic/order-service.js:54]: Could you please keep the function name without the byId as was before?
```suggestion
module.exports.getOrder = async function (id) {
```

**Review by rluvaton — APPROVED** [review]
Thanks!

File: example-application/business-logic/order-service.js (hunk 1 above)
Comment (rubengmurray) [reply example-application/business-logic/order-service.js]: done 👍

**Comment by rubengmurray** [comment]
I messed up a merge conflict in package-lock which I've reverted.. only changes in this PR are the method name updates from user > order

**Comment by mikicho** [comment]
Great catch!

**Pull request merged**